WalletWasabi.Fluent/ViewModels/AddWallet/RecoverWalletViewModel.cs

```diff
@@ -85,7 +86,7 @@ private async Task OnNextAsync(string? walletName)
 						return result;
 					});
 
-				Navigate().To(new AddedWalletPageViewModel(keyManager));
+				await NavigateDialogAsync(new CoinJoinProfilesViewModel(keyManager, true), NavigationTarget.DialogScreen);
```

Comment:
Navigation target is not needed. It will take it automatically.
```suggestion
				await NavigateDialogAsync(new CoinJoinProfilesViewModel(keyManager, isNewWallet: true));
```